temporal/t.rast3d.univar/testsuite/test_t_rast3d_univar.py

```diff
@@ -162,6 +169,43 @@ def test_6_error_handling_no_input(self):
         # No input
         self.assertModuleFail("t.rast3d.univar", output="out.txt")
 
+    def test_7(self):
```

Comment:
Let's not continue the bad practice of numbering the functions instead of naming them.

Reply:
Right.

Reply:
Adressed in [5e8c340](https://github.com/OSGeo/grass/pull/2588/commits/5e8c340160734e60a8f506f2563f25fd882f06ca) and [f6b9729](https://github.com/OSGeo/grass/pull/2588/commits/f6b972962c678a042d8152cc5b92075a4fcf92f4)